### PR TITLE
Added info about apache_tika_path parameter related to BinaryFile Field indexing

### PR DESCRIPTION
**Pull request by vidarl**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | N/A
| Versions      | 3.3, 4.0

Added information about related setting when configuring BinaryFile Field indexing

### CODE REVIEW

File: docs/guide/file_management.md [hunk 1]
```diff
@@ -139,6 +139,8 @@ The following file types are indexed by default:
 - application/vnd.openxmlformats-officedocument.spreadsheetml.sheet
 ```
 
+The default path to the Tika jar is specified with the `apache_tika_path` parameter in `config/packages/ezcommerce/ecommerce_parameters.yaml`
```
Comment (juskora) [inline docs/guide/file_management.md:142]:
```suggestion
The default path to the Tika jar is specified with the `apache_tika_path` parameter in `config/packages/ezcommerce/ecommerce_parameters.yaml`.
```

**Review by juskora — APPROVED** [review]

**Comment by juskora** [comment]
@vidarl does it also apply to v. 4.1?

**Comment by vidarl** [comment]
> @vidarl does it also apply to v. 4.1?

yes, but now I notice there is a small difference in 4.0 and 4.1 vs 3.3
In 3.3, the path is `config/packages/ezcommerce/ecommerce_parameters.yaml`
in 4.0 and 4.1 the path is `config/packages/commerce/commerce_parameters.yaml`

**Comment by juskora** [comment]
> > @vidarl does it also apply to v. 4.1?
> 
> yes, but now I notice there is a small difference in 4.0 and 4.1 vs 3.3 In 3.3, the path is `config/packages/ezcommerce/ecommerce_parameters.yaml` in 4.0 and 4.1 the path is `config/packages/commerce/commerce_parameters.yaml`

thanks for the info, I've changed it respectively.

**Pull request merged**